specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/containers.json

```diff
@@ -0,0 +1,449 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2015-12-01-preview",
+        "title": "StorageManagementClient",
+        "description": "The Admin Storage Management Client."
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage.Admin/farms/{farmId}/shares/operationresults/{operationId}": {

```

Comment:
How do we get the operation id?

Reply:
It is the JobId in the result from starting a container migration.

Reply:
got it, found it here https://github.com/deathly809/azure-rest-api-specs/blob/ce60ea4167ea0298fee6c3f67a90e1c98b5ffda0/specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/containers.json#L303